test/scripts/Dockerfile.tests

```diff
@@ -1,14 +1,16 @@
-FROM node:latest
+FROM node:alpine
+
+RUN apk update && apk add curl
 
 WORKDIR /usr/src/app
-RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.16.0/bin/linux/amd64/kubectl && \
+RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/`curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt`/bin/linux/amd64/kubectl && \
```

Comment:
I don't agree with this change; the version should be pinned so that this katapult version is always the same.

Reply:
I've removed the change